### ISSUE TITLE
Multiply feature matrix by 100?

### ISSUE THREAD

**Issue by catalys1** [GuYuc/WS-DAN.PyTorch#23]
In the forward pass of the model ([here](https://github.com/GuYuc/WS-DAN.PyTorch/blob/master/models/wsdan.py#L110)) we have this line, which calculates the class logits:

```python
# Classification
p = self.fc(feature_matrix * 100.)
```

I'm not sure where this multiply by 100 magic number is coming from. Can you tell me why this is here, and why it's necessary? When I remove it, learning seems to stall. The only thing I can think is that it's supposed to boost the gradient, I'm just not sure why this is necessary.

**Comment by GuYuc** [comment]
Hi. This idea was copied from the authors, they explained that such step would help accelerate the training process. See this issue #1 for details.
